packages/tables/resources/views/index.blade.php

```diff
@@ -749,7 +750,7 @@ class="filament-table-header-cell-{{ \Illuminate\Support\Str::of($column->getNam
                                     </x-tables::actions.cell>
                                 @endif
 
-                                @if ($isSelectionEnabled)
+                                @if ($isSelectionEnabled && $recordBulkActionEnabled)
```

Comment:
Maybe this can be nested. No need to evaluate `$recordBulkActionEnabled()` if `$isSelectionEnabled()` is false.

Reply:
i don't see how to change this.

Reply:
You can nest this and run `$isRecordBulkSelectable` only if `$isSelectionEnabled` is true.